src/walberla_bridge/src/lattice_boltzmann/lb_kernels.hpp

```diff
@@ -53,14 +59,17 @@ template <typename FT = double, Arch AT = Arch::CPU> struct KernelTrait {
       pystencils::CollideSweepDoublePrecisionThermalizedAVX;
   using CollisionModelLeesEdwards =
       pystencils::CollideSweepDoublePrecisionLeesEdwardsAVX;
+  using StreamSweep = pystencils::StreamSweepDoublePrecisionAVX;
```

Comment:
The AVX stream sweeps only improve performance in single-precision. I'm including the double-precision kernel to be consistent.